Source/ASDisplayNode.mm

```diff
@@ -284,7 +284,7 @@ + (void)initialize
 }
 
 #if !AS_INITIALIZE_FRAMEWORK_MANUALLY
-+ (void)load
+__attribute__((constructor)) static void ASLoadFrameworkInitializer(void)
 {
   ASInitializeFrameworkMainThread();
```

Comment:
Agree that using static attribute initializer is the best option here.

We should update the documentation of `-[ASConfigurationDelegate textureDidInitialize]` to state that it's called pre-main instead of +load.

@Adlai-Holler @maicki Do you know if `-textureDidInitialize` is useful for any clients? If it can be removed, and if we can shut down `ASExperimentalLayerDefaults` experiment (i.e gather `allowsGroupOpacityFromUIKitOrNil` and `allowsEdgeAntialiasingFromUIKitOrNil` on demand), then we can remove `ASInitializeFrameworkMainThread()` completely.

Reply:
Thanks for catching that, @nguyenhuy! I noticed that comment when I was debugging this but forgot to update it with this patch. I'll push an update in a minute.

Reply:
Awesome, thanks!

Reply:
We do rely on `-textureDidInitialize` currently so I don't want to get rid of it right now but let me think of how we can remove our reliance on it.